specification/cognitiveservices/OpenAI.Inference/models/completions/chat_completions.tsp

```diff
@@ -2,6 +2,7 @@ import "@typespec/rest";
 import "@typespec/http";
 import "@typespec/versioning";
 
+import "./streaming/chat_completions.tsp";
```

Comment:
Can you help me understand the purpose of `chat_completions` including `/streaming/chat_completions.tsp`  but why not `/streaming/chat_completions.tsp` include `chat_completions`?

Reply:
This import is so that this code in the `tsp` file gets picked up by the TSP compiler, otherwise it's never reached. The models under the `streaming` are not meant to be exposed by operation (only by the `client.tsp`) therefore, they never get "organically" imported for usage.